src/test_utils.rs

```diff
@@ -111,10 +111,10 @@ impl TestedDialects {
     /// that:
     ///
     /// 1. parsing `sql` results in the same [`Statement`] as parsing
-    /// `canonical`.
+    ///     `canonical`.
```

Comment:
rustfmt 1.80.0 requires an indent before the next line, so just fix it to pass the CI. #1352 also did this.

Reply:
I opened a separate PR for this: https://github.com/sqlparser-rs/sqlparser-rs/pull/1357